src/classes/application.js

```diff
@@ -12,22 +13,35 @@ module.exports = class Application extends Base {
     const config = require(path.join(process.cwd(), 'config/environment.js'))(process.env.NODE_ENV)
     const MiddlewareRouter = require(path.join(process.cwd(), 'app/middleware.js'))
     const Router = require(path.join(process.cwd(), 'app/router.js'))
+    const log = new Log()
```

Comment:
yea i think it probably should. Thought about that at the time but was in a "just get it working" mood at that time. Time to revisit